packages/components/pie-card/test/component/pie-card.spec.ts

```diff
@@ -304,4 +304,31 @@ test.describe('PieCard - Component tests', () => {
             });
         });
     });
+
+    test.describe('Prop: disabled', () => {
+        test.describe('when an image exists and the prop `disabled` is set to `true`', () => {
```

Comment:
For the sake of coverage we probably need a corresponding test for when disabled is false too, now I think of it

Reply:
Added.